hs_core/templates/pages/genericresource.html

```diff
@@ -975,6 +989,78 @@ <h4 class="modal-title">Confirm file deletion</h4>
                                 </div>
                                 {% endif %}
 
+                                  <!-- ========= CONNECTIONS TAB ========= -->
+                                {% if connections %}
+                                    <div class="tab-pane" id="connectionsTab">
+                                         {% if connections.collected_by %}
+                                                <h4><strong>Belongs to Collections:</strong></h4>
+                                                <hr style="margin-top: 0;margin-bottom: 2px">
+                                                <table id="collected-by-table" class="table-hover table-striped resource-custom-table" width="100%">
+                                                    <thead>
+                                                        <tr>
+                                                            <th>Title</th>
+                                                            <th>Owners</th>
+                                                            <th>Sharing Status</th>
+                                                            <th>My Permission</th>
+                                                        </tr>
+                                                    </thead>
+                                                    <tbody>
+                                                        {% for res in connections.collected_by %}
```

Comment:
Refer to my 8/31/16 email when we were discussing collections and the comments in #1403 for the context of the "connections" suggestion.  The use case that we are striving to achieve is that the landing page of a member resource have the ability to display the collections it belongs to.  But an "isPartOf relationship cannot be implemented because the user creating the collection does not have the permissions to change the member resource.  So I suggested that the connections tab be used to display dynamic connections between resources that are represented in the Django database but are not part of the resource metadata and not editable by a user.  This separates dynamic connections that the system automatically maintains, but are not in the resource metadata xml, from formal relationships that a user may add.  If there is a clean way to have these in the same tab, I am OK with that too, but separation seemed better at the time I was suggesting this.